components/web/goaccess/goaccess.p5m

```diff
@@ -22,14 +23,17 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=etc/goaccess.conf preserve=true
```

Comment:
This got deleted during the update of the manifest. Please leave existing manual additions in when updating.

Reply:
Please also make sure that config file locations don't change unless absolutely necessary. Existing users will need to be notified about the update otherwise.

Reply:
Thank you Toasterson, it's fixed, now the previous config file is there with preserve=true.